scripts/test-live2d-eye-motion.mjs

```
@@ -0,0 +1,129 @@
+  assert.equal(values.get('ParamEyeLOpen'), 0.8)
+  assert.equal(values.get('ParamEyeROpen'), 0.6)
+
+  finishComposition()
+  motionEyes = [0.4, 0.7]
+  tick(48)
+  assert.equal(values.get('ParamEyeLOpen'), 0.4)
+  assert.equal(values.get('ParamEyeROpen'), 0.7)
+
+  assert.equal(composeLive2DEyeMotion({}, motionManager, facialManager), undefined)
+
+  const fixtureUrl = new URL(
```

ESLint

[error] 'URL' is not defined.

(no-undef)